constellations/orion/src/domains/family_valid_test.go

```diff
@@ -0,0 +1,59 @@
+//ID,password and primary contact email
+
+package domains_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+)
+
+func TestValidPrimaryEmail(t *testing.T) {
```

Comment:
I know it's inconsistent in the codebase, but let's make all test names be:

`Test_ValidPrimaryEmail(...)`
or
`Test_xxxxxx`